client/core/core.go

```diff
@@ -1987,19 +1899,23 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) error {
 	return nil
 }
 
-// connectDEX creates and connects a dexConnection, but does not authenticate
-// the connection through the 'connect' route.
+// connectDEX establishes a ws connection to a DEX server using the provided
+// url and cert, but does not authenticate the connection through the 'connect'
+// route.
 func (c *Core) connectDEX(acctInfo *db.AccountInfo) (*dexConnection, error) {
 	// Get the host from the DEX URL.
-	uri := acctInfo.URL
-	parsedURL, err := url.Parse(uri)
+	parsedURL, err := url.Parse(acctInfo.URL)
 	if err != nil {
-		return nil, fmt.Errorf("error parsing account URL %s: %v", uri, err)
+		return nil, fmt.Errorf("error parsing account URL %s: %v", acctInfo.URL, err)
+	}
+	uri := parsedURL.Host // empty for urls without scheme
+	if uri == "" {
+		uri = parsedURL.String()
```

Comment:
This only seems to help with ""//127.0.0.1:7232", which isn't much of an improvement over a random shceme.  Doesn't seem to work.  If you give it "127.0.0.1:7232" without the scheme, then `url.Parse` errors.  Seems like you'd want to try `net.SplitHostPort` in the event that `Parse` fails. e.g. https://play.golang.org/p/_iJOLLmeyzV  But let's address this later.

```
$   ./dexcctl -u u -P p getfee "http://127.0.0.1:7232" ~/.dcrdex/rpc.cert 
{
  "fee": 100000000
}
$   ./dexcctl -u u -P p getfee "127.0.0.1:7232" ~/.dcrdex/rpc.cert 
error parsing account URL 127.0.0.1:7232: parse "127.0.0.1:7232": first path segment in URL cannot contain colo

Reply:
iirc, I tried with `localhost:35` and that worked.

Reply:
Oh, I think's that's because with `localhost:35` it interprets that in the format `scheme:opaque?query#fragment` so that `URL.String` is printing it as `scheme+":"+opaque`.

Reply:
Yup, it thinks `localhost` is the scheme in an opaque url that way https://play.golang.org/p/2KxZxqeMOTN

Let's fix this in another PR though.